commit 0bb63448bd58fb8e9676cd903c4fc52568ba54e3
Author: Olly Betts <olly@survex.com>
Date:   Thu Dec 12 17:35:45 2024 +1300

    Normalise -0.00 in DXF bounds properly
    
    The existing normalisation worked for coordinates, but introduced
    a space before bounds.

diff --git a/tests/cavern.tst b/tests/cavern.tst
index 39f1cfa3..e004d208 100755
--- a/tests/cavern.tst
+++ b/tests/cavern.tst
@@ -346,7 +346,7 @@ for file in $TESTS ; do
     case $pos in
       dxf)
 	# On x86 excess precision can result in -0.00 for some coordinates.
-	sed 's/-0\.00\>/ 0.00/g' < "$tmpfile" > tmp.tmp
+	sed 's/^-0\.00\>/0.00/;s/ -0\.00\>/  0.00/g' < "$tmpfile" > tmp.tmp
 	mv tmp.tmp "$tmpfile"
 	;;
       json)
